templates/generic/bpir4_reference.json

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
does single file works for both 32 bit and 64 bit

Reply:
Hi @cpokuru ,
As we discussed only generic json file we will be using which will support 64 bit.